test/e2e/utils/nodes/nodes.go

```diff
@@ -40,8 +42,16 @@ const (
 	LabelRole = "node-role.kubernetes.io"
 	// LabelHostname contains the key for the hostname label
 	LabelHostname = "kubernetes.io/hostname"
+
+	TestNodeLabel = "rte-e2e-test-node"
```

Comment:
this constant is used here and in `pods.go`. The two modules should not import each other (if this is already the case, we should fix it in the future). Let's thus move this contant, and possibly others, in another package in a future PR.